### ISSUE TITLE
Vue 3 Support Default for Vue Meta

### ISSUE THREAD

**Issue by Penguinlay** [nuxt/vue-meta#742]
Since Vue 3 is now the default version, I think Vue Meta should be updated accordingly to support Vue 3 as default like all the other Vue plugins.

**Comment by Penguinlay** [comment]
This can still be reproduced.

**Comment by chaderenyore** [comment]
> Since Vue 3 is now the default version, I think Vue Meta should be updated accordingly to support Vue 3 as default like all the other Vue plugins.

Yeah I agree. I really Need it now, and it is still in alpha so I can't use it in production. I can't just switch to nuxt entirely now.

**Comment by YusufcanY** [comment]
> > Since Vue 3 is now the default version, I think Vue Meta should be updated accordingly to support Vue 3 as default like all the other Vue plugins.
> 
> Yeah I agree. I really Need it now, and it is still in alpha so I can't use it in production. I can't just switch to nuxt entirely now.

Totally agree. Please update the package. It's in alpha still.

**Comment by Penguinlay** [comment]
If there is no active maintainers for this, I am interested in taking over!
